### ISSUE TITLE
Is there a difference result between use_cat ?

### ISSUE THREAD

**Issue by beandkay** [microsoft/Semi-supervised-learning#18]
Hi,

Just wondering are there any differences on the result between model with use_cat True and use_cat False?

**Comment by qianlanwyd** [comment]
When you use resnets, the statistics of batchnorm will be different. When you use transformers, I think there is no difference.

**Comment by beandkay** [comment]
so basically, use_cat True will yield better results for vision tasks?

**Comment by Hhhhhhao** [comment]
> so basically, use_cat True will yield better results for vision tasks?

On classic vision tasks, as the backbone is resnet-based and has batch norm layers, you should always set use_cat as True.
On USB vision tasks where we use vision transformers, the result would be same.

**Comment by beandkay** [comment]
Thank you, that makes sense.
